content/doc/book/system-administration/security/build-authorization.adoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: documentation
+---
+
+== Access Control for Builds
+
+Similar to access control for users, builds in Jenkins run with an associated user authorization.
+By default, builds run as the internal SYSTEM user that has full permissions to run on any node, create or delete jobs, start and cancel other builds, etc.
+
+NOTE: The permission Agent/Build requires access control for builds to be set up, as the build's authentication is checked, and not the user starting the build.
+
+In a Jenkins setup with fine-grained permissions control, this is undesirable.
+For example, having builds run as SYSTEM could allow users with access to configure and start one job to start builds of any other jobs using plugin:pipeline-build-step[Pipeline Build Step Plugin].
+
+NOTE: Some plugins implement their own access control beyond build authorization.
+Not using build authorization, or having builds run as SYSTEM, is an _indicator_ that problems like the one described above can occur.
+
+The solution to this is to configure access control for builds.
+The most notable plugin in this space is plugin:authorize-project[Authorize Project Plugin], which allows flexible configuration of global and per-project build authorization.
```

Comment:
Add a link to https://jenkins.io/doc/developer/extensions/jenkins-core/#queueitemauthenticator for the list of other plugins?

Reply:
There are none. (Different for the core PR, where there's no way to update the message ad-hoc as needed.)